src/cf-typography/src/molecules/pull-quote.less

```diff
@@ -0,0 +1,26 @@
+.m-pull-quote {
+
+    &_body {
+        .heading-3();
+
+        color: @pull-quote;
+
+        .respond-to-max( @bp-xs-max, {
+            .heading-4();
+        } );
+    }
+
+    &_citation {
+        .heading-5();
+
+        color: @pull-quote_citation;
+    }
+
+    &__large .m-pull-quote_body {
+        .heading-2();
+
+        .respond-to-max( @bp-xs-max, {
+            .heading-3();
```

Comment:
> [W]hy add extra styles if we don't want them?

In order to simplify the experience of developing _this_ code. I am for the goal of slimming down output, but it has to be balanced with the goal of a good developer experience.

In this case, it's not making this code _so_ much more complex, so I will concede that it's probably worth it, in this case.